arches/app/templates/views/components/cards/default.htm

```diff
@@ -367,20 +367,30 @@ <h4 class="card-summary-name">
             <!-- /ko -->
             {% block form_buttons %}
             <div class="install-buttons">
+                <!-- ko if: !!ko.unwrap(card.altButtons) -->
+                    <!-- ko foreach: {data: card.altButtons, as: 'button'} -->
+                    <button data-bind="text: button.label, click: button.onClick($parent), attr: {'class': button.classes}, css: {disabled: (!$parent.card.isWritable && !self.preview)}, visible: button.visible($parent)" ></button>
```

Comment:
`css: {disabled: (!$parent.card.isWritable && !self.preview)}` seems problematic to me - I don't think we always want to disable additional buttons based on these conditions.  We should probably allow a user to define when a button should be disabled themselves.

I'm uncertain what this is: `visible: button.visible($parent)`; if we want to be able to control visibility of individual buttons, shouldn't this just be an observable?